LightboxOverlay.js

```diff
@@ -200,24 +200,70 @@ var LightboxOverlay = React.createClass({
         </TouchableOpacity>
       )
     )}</Animated.View>);
-    var content = (
-      <Animated.View style={[openStyle, dragStyle]} {...handlers}>
-        {this.props.children}
-      </Animated.View>
-    );
-    if(this.props.navigator) {
-      return (
-        <View>
-          {background}
-          {content}
-          {header}
-        </View>
+
+    function onViewTransformed({translateY}) {
```

Comment:
Why do you have two `onViewTransformed` and `onTransformGestureReleased` functions? It seems that they are identical...